hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxyImpl.java

```diff
@@ -333,6 +334,7 @@ public boolean isLocked(K key) {
 
     @Override
     public Map<K, V> getAll(Set<K> keys) {
+        checkNotNull(keys, NULL_KEYS_ARE_NOT_ALLOWED);
```

Comment:
That is right, I didn't look into `CollectionUtil.isEmpty(keys)`, it also checks for nulls, which I wasn't aware... The name is misleading though, should be `isNullOrEmpty`...
In this case I need to drop this check for backward-compatibility. Good catch!